auth-web/src/components/auth/staff/account-management/StaffRejectedAccountsTable.vue

```diff
@@ -1,50 +1,108 @@
 <template>
-  <v-data-table
-    class="user-list"
-    :headers="headerAccounts"
-    :items="rejectedTasks"
-    :items-per-page.sync="tableDataOptions.itemsPerPage"
-    :custom-sort="columnSort"
-    :no-data-text="$t('noActiveAccountsLabel')"
-    :footer-props="{
-        itemsPerPageOptions: getPaginationOptions
-      }"
-    :options.sync="tableDataOptions"
-    @update:items-per-page="saveItemsPerPage"
-    :server-items-length="totalRejectedTasks"
-  >
-    <template v-slot:loading>
-      Loading...
-    </template>
-    <template v-slot:[`item.dateSubmitted`]="{ item }">
-      {{formatDate(item.dateSubmitted, 'MMM DD, YYYY')}}
-    </template>
-    <template v-slot:[`item.action`]="{ item }">
-        <v-btn
-          outlined
-          color="primary"
-          class="action-btn"
-          :data-test="getIndexedTag('reset-password-button', item.id)"
-          @click="view(item)"
-        >
-          View
-        </v-btn>
-    </template>
-  </v-data-table>
+  <v-container class="pa-0">
+    <DatePicker v-show="showDatePicker" ref="datePicker" @submit="updateDateRange($event)" />
+    <v-form class="fas-search account-rejected-search">
+      <v-row dense class="row-margin">
+        <v-col sm="12" cols="6">
+          <transition name="slide-fade">
+            <v-data-table class="user-list" :headers="headerAccounts" :items="rejectedTasks"
+              :items-per-page.sync="tableDataOptions.itemsPerPage" :custom-sort="columnSort"
```

Comment:
might want to fix the formatter, so it puts attributes on one line each like the code before, otherwise it makes it hard to see the difference